api/controllers/employees.js

```diff
@@ -0,0 +1,86 @@
+const sequelize = require('sequelize');
+
+const logger = require('../logger');
+const Employee = require('../models/employee');
+const Transaction = require('../models/transaction');
+
+const EmployeeController = {};
+
+EmployeeController.getEmployees = async (req, res, next) => {
+    try {
+        const employees = await Employee.findAll();
+        res.json(employees);
+    } catch(err) {
+        res.status(500).send("An error occurred when getting employees.");
```

Comment:
so catch blocks will no longer send http responses but instead will throw errors that will be caught by express's error handler. we can use https://github.com/jshttp/http-errors